src/components/address-text/address-text.tsx

```diff
@@ -74,38 +74,38 @@ export function AddressText({
         containerProps={{ ref: copyToClipboardRef }}
       />
     </HStack>
-  );
+  )
 }
 
 export function useAddressText(i: Identity | string) {
   const q = useQuery({
     queryKey: ["address", i],
     queryFn: async () => {
       if (i instanceof Identity) {
-        return (await i.getAddress()).toString();
+        return (await i.getAddress()).toString()
       }
-      return typeof i === "string" ? i : "";
+      return typeof i === "string" ? i : ""
     },
-  });
-  return q?.data ?? "";
+  })
+  return q?.data ?? ""
 }
 
-const ELLIPSIS_SIZE = 2;
+const ELLIPSIS_SIZE = 2
 export function shortenAddressText(
   offsetWidth: number,
   scrollWidth: number,
-  addressText: string
+  addressText: string,
 ): string {
-  const isValid = validateAddress(addressText);
-  if (isValid !== true) throw new Error(isValid);
-  if (offsetWidth >= scrollWidth) return addressText;
-  const addressLen = addressText.length;
-  const halfLen = Math.round(addressLen / 2);
-  const avgCharSize = Math.round(scrollWidth / addressLen);
+  const isValid = validateAddress(addressText)
+  if (isValid !== true) throw new Error(isValid)
+  if (offsetWidth >= scrollWidth) return addressText
+  const addressLen = addressText.length
+  const halfLen = Math.round(addressLen / 2)
+  const avgCharSize = Math.round(scrollWidth / addressLen)
   const charsToRemove =
     Math.round((scrollWidth - offsetWidth) / Math.round(avgCharSize)) +
-    ELLIPSIS_SIZE;
-  const first = addressText.slice(0, halfLen - charsToRemove / 2);
-  const second = addressText.slice(halfLen + charsToRemove / 2);
-  return first + "..." + second;
+    ELLIPSIS_SIZE
+  const first = addressText.slice(0, halfLen - charsToRemove / 2)
+  const second = addressText.slice(halfLen + charsToRemove / 2)
+  return first + "..." + second
```

Comment:
All the other changes in this file (and others) are because `.prettierrc` is set to strip semi-colons.